src/components/Form/Form.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { IoMdCheckmark } from "react-icons/io";
+
+const Form = () => {
+  return (
+    <form className="search-form">
+      <div className="search-form--specification">
+        <label for="ingredient" className="search-form__label">
+          Ingredient:
+        </label>
+        <input
+          id="ingredient"
+          className="search-form__input"
+          placeholder="eggs, spinach"
+          type="text"
+        />
+        <ul className="search-form--ingredient-list"></ul>
```

Comment:
This list will appear after the user click submit button, correct?

Reply:
yes, correct